src/assignment/assignment.controller.ts

```diff
@@ -28,6 +36,10 @@ export class AssignmentController {
         throw new Error('Must be one of archive, reviewed, to-review, upcoming');
       }
 
+      if (email) {
```

Comment:
It seems like an authenticated user can load another user's assignments just by knowing their email? Is that okay?

Reply:
@godfreyyeung yep!